gui/src/3D/controls/holographicSlate.ts

```diff
@@ -366,24 +330,24 @@ export class HolographicSlate extends ContentDisplay3D {
 
     protected _affectMaterial(mesh: AbstractMesh) {
         // TODO share materials
-        this._backPlateMaterial = new FluentBackplateMaterial(`${this.name} plateMaterial`, mesh.getScene());
+        this._titleBarMaterial = new FluentBackplateMaterial(`${this.name} plateMaterial`, mesh.getScene());
 
         this._pickedPointObserver = this._host.onPickedPointChangedObservable.add((pickedPoint) => {
             // if (pickedPoint) {
-            //     this._backPlateMaterial. = pickedPoint;
-            //     this._backPlateMaterial.hoverColor.a = 1.0;
+            //     this._titleBarMaterial.globalLeftIndexTipPosition = pickedPoint;
+            //     this._titleBarMaterial.hoverColor.a = 1.0;
             // } else {
-            //     this._backPlateMaterial.hoverColor.a = 0;
+            //     this._titleBarMaterial.hoverColor.a = 0;
             // }
         });

```

Comment:
if nothing is actually running here, maybe you can either comment the entire `add` function or remove it altogether.